src/_includes/blocks/footer.njk

```diff
@@ -1,67 +1,69 @@
+
 <div id="footer" class="text-sm mt-4 mx-4 mb-0">
   The Globe Church is all about Jesus, made up of all sorts of people, involved in the greatest mission...
 </div>
 
-<footer id="footer" class="border-t text-sm pt-4 mt-4 mx-4">
-<!--
+<footer id="footer" class="border-t text-sm pt-4 mt-4 mx-4 md:flex md:items-end md:justify-between">
   <div>
-    {{ meta.registeredAddress}}
-  </div>
- -->
-  <nav>
-    <div class="grid grid-cols-2 sm:grid-cols-3 md:grid-cols-6 lg:grid-cols-6 gap-5 md:max-w-[75%]">
-      <ul>
-        <li><a class="text-base " href="/about">About</a></li>
-        <li><a  href="/the-globe-church">The Globe Church</a></li>
-        <li><a  href="/the-team">The team</a></li>
-        <li><a  href="/what-we-believe">What we believe</a></li>
-        <li><a  href="/affiliations">Affiliations</a></li>
-        <li><a  href="/jobs">Jobs</a></li>
-        <li><a  href="/policies">Policies</a></li>
-      </ul>
+    <nav>
+      <div class="grid grid-cols-2 sm:grid-cols-3 md:grid-cols-6 lg:grid-cols-6 gap-5 md:max-w-[75%]">
+        <ul>
+          <li><a class="text-base " href="/about">About</a></li>
+          <li><a  href="/the-globe-church">The Globe Church</a></li>
```

Comment:
In all these links there is a weird extra space between the `a` and the first property (eg `href`). Would be good to resolve that for all of the items.